libs/vkd3d/command.c

```diff
@@ -5979,33 +5975,31 @@ static void d3d12_command_list_copy_image(struct d3d12_command_list *list,
             goto cleanup;
         }
 
-        extent.width = d3d12_resource_desc_get_width(&dst_resource->desc, dst_view_desc.miplevel_idx);
-        extent.height = d3d12_resource_desc_get_height(&dst_resource->desc, dst_view_desc.miplevel_idx);
-        extent.depth = dst_view_desc.layer_count;
+        memset(&attachment_info, 0, sizeof(attachment_info));
+        attachment_info.sType = VK_STRUCTURE_TYPE_RENDERING_ATTACHMENT_INFO_KHR;
+        attachment_info.imageView = dst_view->vk_image_view;
+        attachment_info.imageLayout = dst_layout;
+        attachment_info.loadOp = VK_ATTACHMENT_LOAD_OP_LOAD;
```

Comment:
Ideally we'd use DONT_CARE and restrict renderArea to copy dest region to be friendlier to hypothetical TBDR.

Reply:
Done.